packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -137,15 +190,31 @@ The content of the `HistoryQuery` component can be overridden. For replacing the
 content, the `suggestion-content` slot is available, containing the history query suggestion (in the
 `suggestion` property), and the matching query part HTML (in the `queryHTML` property).
 
-```vue
-<HistoryQueries>
-  <template #suggestion-content="{ queryHTML }">
-    <img class="x-history-query__history-icon" src="./history-icon.svg"/>
-    <span class="x-history-query__matching-part" v-html="queryHTML"></span>
-  </template>
-  <template #suggestion-remove-content>
-    <img class="x-history-queries__remove" src="./remove-icon.svg"/>
-  </template>
-</HistoryQueries>
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work